tests/topotests/bgp_snmp_mplsl3vpn/test_bgp_snmp_mplsvpn.py

```diff
@@ -253,19 +253,19 @@ def _convergence():
 
 
 interfaces_up_test = {
-    "mplsL3VpnConfiguredVrfs": "2",
-    "mplsL3VpnActiveVrfs": "2",
-    "mplsL3VpnConnectedInterfaces": "3",
+    "mplsL3VpnConfiguredVrfs": "3",
+    "mplsL3VpnActiveVrfs": "3",
+    "mplsL3VpnConnectedInterfaces": "10",
```

Comment:
There is not always gre0, gretap, erspan interfaces.

When there is no gre0, gretap, erspan, the number is 7

Reply:
add a patch to account for the real number of interfaces in the default vrf.